pkg/controller/rotation/handle_verification_rotate.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rotation
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/google/exposure-notifications-server/pkg/logging"
+	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/google/exposure-notifications-verification-server/pkg/observability"
+	"github.com/hashicorp/go-multierror"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+)
+
+// HandleVerificationRotate handles verification certificate key rotation.
+func (c *Controller) HandleVerificationRotate() http.Handler {
+	type Result struct {
+		OK     bool    `json:"ok"`
+		Errors []error `json:"errors,omitempty"`
```

Comment:
wait can we render `[]error` rather than `[]string`. I think Seth just did something with this

Reply:
https://github.com/google/exposure-notifications-verification-server/pull/1613

Reply:
yeah, that needs a rebase...